person.rb

```diff
@@ -0,0 +1,31 @@
+class Person < Nameble
+  attr_reader :id, :rentals, :name, :age
+
+  def initialize(age, name = 'Unknown', permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = permission
+    @rental = []
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def correct_name
+    @name
+  end
+
+  def add_rental(rental)
+    @rental.push(rental)
+    rental.person = self
+  end
+
```

Comment:
- [ ] The previous reviewer suggested you implement the `add_rental` method using the same pattern as you have taken in the Book class, but in this case `book` and `date` are the parameters.

  ```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  ```